packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -487,15 +491,25 @@ export class SDKClient {
         }
     };
 
-    private captureMetrics = (mode, type, status, cost, caller, interactingEntity) => {
+    private captureMetrics = (mode, type, status, cost, gas, caller, interactingEntity) => {
         const resolvedCost = cost ? cost : 0;
+        const resolvedGas = typeof gas === 'object' ? gas.toInt() : 0;
         this.consensusNodeClientHistorgram.labels(
             mode,
             type,
             status,
             caller,
-            interactingEntity)
+            interactingEntity,
+            'cost')
             .observe(resolvedCost);
+        this.consensusNodeClientHistorgram.labels(
+            mode,
+            type,
+            status,
+            caller,
+            interactingEntity,
+            'gas')
```

Comment:
same as above, if 'gas' is a type, it should be sent on the already type.
But since we have already a use-case for type (Transaction and Query) probably we would want to create a new metric for gas capturing.

When I mean a new metric I mean a new:
 `const metricHistogramName = 'rpc_relay_consensusnode_response';
        register.removeSingleMetric(metricHistogramName);
        this.consensusNodeClientHistorgram = new Histogram({`
        
Name could be: `rpc_relay_consensusnode_gasfee`

Reply:
Yeah, i think a separate metric would be cleaner.
If any aggregation is done without filtering for labels the values will be off.
Prior to this all observed values would be for hbar transaction cost.
As coded it'll mix hbar cost and gas charged.